nmmo/core/env.py

```diff
@@ -366,7 +366,8 @@ def _compute_observations(self):
 
       obs[agent_id] = Observation(
         self.config, self.realm.tick,
-        agent_id, visible_tiles, visible_entities, inventory, market)
+        agent_id, visible_tiles, visible_entities, inventory, market,
+        agent.in_combat)
```

Comment:
i don't think we want to pass this directly to obs, shoudn't this just be a column on EntityState?

Reply:
`latest_combat_tick` is added to EntityState, and `Observation` computes whether this agent is in combat status. Other places have been modified accordingly.